app/controllers/api/concerns/act_as_api_request.rb

```diff
@@ -24,7 +24,7 @@ def render_error(status, message, _data = nil)
         response = {
           error: message
         }
-        render json: response, status: status
+        render json: response, status:
```

Comment:
😮 I'll need to get used to this ruby 3.1 syntax

Reply:
Is this mandatory?

Reply:
rubocop has reported it @sebastiancaraballo, but no, it's not mandatory.